modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -52,6 +52,7 @@
         :server-error="fields[field.name].serverError"
         :doc-id="docId"
         :generation="generation"
+        @update:model-value="updateNextAndEmit"
```

Comment:
With this, no more need for a deep watcher.

Reply:
To keep the behavior consistent, `fieldState` was watched before, I just replaced the watch by this. `compareMetaState` wasn't.